File: backend/PizzaDeliveryBackend/Services/DrinkService.cs
using PizzaDeliveryBackend.Models;

namespace PizzaDeliveryBackend.Services
{
    public class DrinkService : BaseService<Drink>, IDrinkService
    {

        MySQLDatabaseContext _context;
        public DrinkService(MySQLDatabaseContext context) : base(context)
        {
            _context = context;
        }
    }
}

File: frontend/src/app/services/drink.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Drink } from '../models/drink.model';
import { BaseService } from './base-service.service';

@Injectable({
  providedIn: 'root'
})
export class DrinkService extends BaseService<Drink> {



  constructor(override http: HttpClient) {
    super(http);
    this.setUrl('drink');
  }

}
